pytype execution fails when using install_all_and_run_tests.sh

I ran $ ./scripts/install_all_and_run_tests.sh (as per the maintainers guide) and received:

```
ninja: Entering directory `.pytype'
[101/192] check slack_bolt.middleware.async_custom_middleware
FAILED: /Users/maria.mercado/bolt-python/.pytype/pyi/slack_bolt/middleware/async_custom_middleware.pyi 
/Users/maria.mercado/bolt-python/env_3.8.5/bin/python3 -m pytype.main --imports_info /Users/maria.mercado/bolt-python/.pytype/imports/slack_bolt.middleware.async_custom_middleware.imports --module-name slack_bolt.middleware.async_custom_middleware --platform darwin -V 3.9 -o /Users/maria.mercado/bolt-python/.pytype/pyi/slack_bolt/middleware/async_custom_middleware.pyi --analyze-annotated --nofail --quick /Users/maria.mercado/bolt-python/slack_bolt/middleware/async_custom_middleware.py
File "/Users/maria.mercado/bolt-python/slack_bolt/middleware/async_custom_middleware.py", line 45, in async_process: Function <callable> expects 1 arg(s), got 0 [wrong-arg-count]
         Expected: (_)
  Actually passed: ()

For more details, see https://google.github.io/pytype/errors.html#wrong-arg-count
ninja: build stopped: subcommand failed.
Leaving directory '.pytype'
```
i made changes to several README and .md files (working on issue #777) but @zimeg also ran the test script and got the same error

--- a/scripts/install_all_and_run_tests.sh
+++ b/scripts/install_all_and_run_tests.sh
@@ -36,6 +36,6 @@ else
     pip install -U pip click && \
     black slack_bolt/ tests/ && \
     pytest && \
-    pip install -U pytype && \
+    pip install "pytype==2022.12.15" && \
     pytype slack_bolt/
 fi

